apps/settings/js/panels/improveBrowserOS/panel.js

```diff
@@ -0,0 +1,34 @@
+define(function(require) {
+  'use strict';
+
+  var SettingsPanel = require('modules/settings_panel');
+
+  return function ctor_improveBrowserOS() {
+    return SettingsPanel({
+      onInit: function(panel) {
+        this.openAppStats = panel.querySelector('#openAppStats');
+        this.openAppStats.addEventListener('click',
+                                          this._onOpenAppStats.bind(this));
+      },
+
+      onUninit: function() {},
```

Comment:
There are default functions in `SettingsPanel` so these three functions can be removed.